src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs

```diff
@@ -993,6 +994,7 @@ private void RegisterEntities()
             RegisterUnit<TrainingKillSwitch>();
             RegisterUnit<Gate>();
             RegisterUnit<RandomRiftPortal>();
+            RegisterUnit<StrongholdExitRift>(); // TODO new exit rift
```

Comment:
There's still a TODO here? We shouldn't be merging anything in that still has work remaining to complete.

Reply:
Yeah bad habit from early in the project where we weren't 100% on proper placement in the bootstrapper